src/xcode/ENA/ENA/Source/Services/CoronaTestService/CoronaTestService.swift

```diff
@@ -130,6 +131,8 @@ class CoronaTestService {
 					Log.info("[CoronaTestService] PCR test registered: \(private: String(describing: self?.pcrTest), public: "PCR Test result")", log: .api)
 
 					Analytics.collect(.testResultMetadata(.registerNewTestMetadata(Date(), registrationToken, .pcr)))
+					// updating badge count for home tab
+					self?.unseenTestsCount.value += 1
```

Comment:
Can we add a parameter `markAsUnseen` to this func so we don't add a badge count if the user adds their test from the home screen? (We have the same thing for health certificates already in place)

Reply:
Make sense